iceoryx_hoofs/container/include/iox/detail/vector.inl

```diff
@@ -359,13 +402,26 @@ inline bool vector<T, Capacity>::erase(iterator position) noexcept
         // AXIVION Next Line AutosarC++19_03-M5.0.9 : False positive. Pointer arithmetic occurs here.
         uint64_t index{static_cast<uint64_t>(position - begin())};
         uint64_t n{index};
-        while ((n + 1U) < size())
+        if constexpr (std::is_trivially_copyable<T>::value)
+        {
+            if constexpr (!(std::is_trivial<T>::value && std::is_class<T>::value))
+            {
+                at(n).~T();
+            }
```

Comment:
I think this can be removed or maybe the check could be for `!is_trivially_destructible`

Reply:
done!